tests/vision/dataset_readers/flickr30k_test.py

```diff
@@ -94,14 +94,14 @@ def test_evaluation_read(self):
         question_tokens = [t.text for t in instance["caption"][0]]
         assert question_tokens == [
             "girl",
-            "without",
+            "with",
             "brown",
             "hair",
             "sits",
             "on",
             "edge",
             "of",
-            "cement",
+            "concrete",
```

Comment:
What are these changes for?

Reply:
Some seeds would select a random "wrong" caption, and then accidentally pick the same caption, because the 3 caption files were the same. Then the test would fail because it would compare two captions that should be different, but weren't. I made a few small tweaks to fix it.